lm_eval/api/task.py

```
@@ -931,10 +977,35 @@ def __init__(
+            if parquet_ds is not None:
+                self.dataset = parquet_ds
+                return
+
+            existing = kwargs.get("download_config")
+            if existing is None:
+                dc = DownloadConfig(local_files_only=True)
+            else:
+                try:
+                    dc = copy(existing)
+                    dc.local_files_only = True
+                except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)